src/components/ExpandedNode/__tests__/ExpandedNode.test.tsx

```diff
@@ -34,7 +34,7 @@ const testDoc: DocumentationResource = {
     ]
   }
 };
-const testActionState: GuidanceState = {
+const testActionState: ActionNode = {
```

Comment:
```suggestion
const testActionNode: ActionNode = {
```

Reply:
Also would need to be changed on lines 104 and 106 where it is used (GH won't let me comment there)